teamA/jeongjaeha/services/postServices.js

```diff
@@ -0,0 +1,50 @@
+const postDao = require('../models/postDao')
+const viewAllDao = require('../models/postDao')
+const viewUserDao = require('../models/postDao')
+const postDeleteDao = require('../models/postDao')
+const postUpdateDao = require('../models/postDao')
+const postLikeDao = require('../models/postDao')
```

Comment:
아래에서 post와 관련된 Dao가 postDao, viewAllDao, ... 등 많이 있습니다.
이를 하나로 통합해서 관리해야 Layered Pattern의 의미가 있습니다.

아래와 같은 구조가 될 수 있도록 수정부탁드립니다~

```suggestion
const postDao = require('../models/postDao)

postDao.create()
postDao.update()
postDao.delete()
postDao.bulk_insert()

```

Reply:
수정하였습니다.